packages/cue/bh/ad/ad.cue

```diff
@@ -384,6 +384,13 @@ ApplicationPolicies: types.#StringEnum & {
 	representation: "applicationpolicies"
 }
 
+IssuancePolicies: types.#StringEnum & {
```

Comment:
Do these new properties have any impact on esc1?

Reply:
These will get ingested regardless but having them defined allows us to display them better in the entity info